### PR TITLE
Adds ru_RU translation

### PR DESCRIPTION
**Pull request by ichorid**
Aside from Russian translation, this PR cleans a lot of language-related stuff in Tribler GUI. Basically, every string in the GUI is now translated. This required refactoring some Channels-related GUI stuff, changing f-strings to `%`-strings, etc.

After we merge this, we should ask translation authors to update their translations.

### CODE REVIEW

**Comment by ichorid** [comment]
retest this please

File: src/tribler-gui/tribler_gui/widgets/lazytableview.py [hunk 1]
```diff
@@ -161,8 +162,11 @@ def on_move_button_clicked(self, _index):
     def resizeEvent(self, _):
         if self.model() is None:
             return
+        viewport_width = self.width()
         for col_num, col in enumerate(self.model().columns):
-            self.setColumnWidth(col_num, self.model().column_width.get(col, lambda _: 110)(self.width()))
+            self.setColumnWidth(
+                col_num, col.width if col.width != EXPANDING else viewport_width - self.model().min_columns_width - 20
```
Comment (devos50) [inline src/tribler-gui/tribler_gui/widgets/lazytableview.py:168]: This change would also better fit in a separate commit.

**Review by devos50 — COMMENTED** [review]
Quite an amount of work 👍

File: src/tribler-gui/tribler_gui/core_manager.py [hunk 2]
```diff
@@ -133,14 +134,14 @@ def should_cleanup_old_versions(self) -> List[TriblerVersion]:
...
         title = "Delete state directories for old versions?"
-        message_body = (
-            f"Press 'Yes' to remove state directories for older versions of Tribler "
-            f"and reclaim {format_size(claimable_storage)} of storage space. "
-            f"Tribler used those directories during upgrades from previous versions. "
-            f"Now those directories can be safely deleted. \n\n"
-            f"If unsure, press 'No'. "
-            f"You will be able to remove those directories from the Settings->Data page later."
-        )
+        message_body = tr(
```
[the remark this answers is not in the document]
Comment (ichorid) [reply src/tribler-gui/tribler_gui/core_manager.py:137]: Sorry to say that, but f-strings are incompatible with `tr()` magic :shrug:

File: src/tribler-gui/tribler_gui/defs.py [hunk 3]
```diff
@@ -132,8 +132,6 @@
 # Tribler shutdown grace period in milliseconds
 SHUTDOWN_WAITING_PERIOD = 120000
 
-ACTION_BUTTONS = 'action_buttons'
```
[the remark this answers is not in the document]
Comment (ichorid) [reply src/tribler-gui/tribler_gui/defs.py:135]: Well, when I started it, it looked quite innocuous...

File: src/tribler-gui/tribler_gui/tribler_window.py [hunk 4]
```diff
@@ -671,7 +676,7 @@ def on_top_search_button_click(self):
         self.has_search_results = True
         self.search_results_page.initialize_root_model(
             SearchResultsModel(
-                channel_info={"name": f"Search results for {query}" if len(query) < 50 else f"{query[:50]}..."},
+                channel_info={"name": (tr("Search results for ") + query) if len(query) < 50 else f"{query[:50]}..."},
```
[the remark this answers is not in the document]
Comment (ichorid) [reply src/tribler-gui/tribler_gui/tribler_window.py:679]: I did most of the changes semi-automatically, with regexps. I'll look at this one.

File: src/tribler-gui/tribler_gui/tribler_window.py [hunk 5]
```diff
@@ -765,11 +770,11 @@ def on_create_torrent(self, checked):
         self.create_dialog.show()
 
     def on_create_torrent_updates(self, update_dict):
-        self.tray_show_message("Torrent updates", update_dict['msg'])
+        self.tray_show_message(tr("Torrent updates"), update_dict['msg'])
 
     def on_add_torrent_browse_file(self, index):
         filenames = QFileDialog.getOpenFileNames(
-            self, "Please select the .torrent file", QDir.homePath(), "Torrent files (*.torrent)"
+            self, tr("Please select the .torrent file"), QDir.homePath(), tr("Torrent files (*.torrent)")
```
[the remark this answers is not in the document]
Comment (ichorid) [reply src/tribler-gui/tribler_gui/tribler_window.py:777]: You are quite right. Again, I did the stuff semi-auto

File: src/tribler-gui/tribler_gui/widgets/channelcontentswidget.py [hunk 6]
```diff
@@ -278,7 +279,7 @@ def reset_view(self, text_filter=None):
         with self.freeze_controls():
             self.controller.table_view.horizontalHeader().setSortIndicator(-1, Qt.DescendingOrder)
         self.model.sort_by = (
-            self.model.columns[self.model.default_sort_column] if self.model.default_sort_column >= 0 else None
+            self.model.columns[self.model.default_sort_column].dict_key if self.model.default_sort_column >= 0 else None
```
[the remark this answers is not in the document]
Comment (ichorid) [reply src/tribler-gui/tribler_gui/widgets/channelcontentswidget.py:282]: Well, the problem is, class properties are not translated by Qt in runtime. The whole column definition thing was using a bunch of class attributes. I could have easily replaced them with instance-level properties, but that would result in repeating translations for *every* subclass (i.e. five times). Also, the thing was a mess and the refactoring was a long-standing necessity.

File: src/tribler-gui/tribler_gui/widgets/lazytableview.py (hunk 1 above)
Comment (ichorid) [reply src/tribler-gui/tribler_gui/widgets/lazytableview.py:168]: Again, that was a necessary part of refactoring the column thing. If you'd like, I can put the channels related changes in a separate commit (in the same PR).

Comment (devos50) [reply src/tribler-gui/tribler_gui/widgets/lazytableview.py:168]: Yeah making a separate commit would be appreciated 👍

**Comment by xoriole** [comment]
@ichorid I agree with @devos50 quite an amount of work. Nice 👍 

But I'd like if it is broken into two separate PRs.
1. Refactoring and required extraction of the strings
2. Russian translations, similar to #6072 neat.

**Comment by ichorid** [comment]
> @ichorid I agree with @devos50 quite an amount of work. Nice +1
> 
> But I'd like if it is broken into two separate PRs.
> 
>     1. Refactoring and required extraction of the strings
> 
>     2. Russian translations, similar to #6072 neat.

Those PRs would be dependent on each other. That would make it very hard to update them. I think just breaking it down into three _commits_ is enough.

**Comment by ichorid** [comment]
retest this please

**Review by devos50 — APPROVED** [review]
I'm fine with having these commits in one PR (in fact, I often make these 'various changes' PRs myself containing commits with different efforts). 👍

**Pull request merged**